README.md

```diff
@@ -1 +1,68 @@
-# space-travelers
\ No newline at end of file
+
+# Getting Started
+
+**[Space travelers]** You can see several pictures with a short description about missions and rockets and you can simulate a making reservation with one of them.
+
+# Built With
```

Comment:
- [ ] Kindly, use the Microverse template for  your Readme file.

Reply:
thanks for your feedback @algerina